sdk/keyvault/azure-keyvault-certificates/tests/test_parse_id.py

```diff
@@ -0,0 +1,42 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# -------------------------------------
+import pytest
+from azure.keyvault.certificates import parse_key_vault_certificate_id
+from _shared.test_case import KeyVaultTestCase
+
+
+class TestParseId(KeyVaultTestCase):
+    def test_parse_certificate_id_with_version(self):
+        # [START parse_key_vault_certificate_id]
+        source_id = "https://keyvault-name.vault.azure.net/certificates/certificate-name/version"
+        parsed_certificate_id = parse_key_vault_certificate_id(source_id)
```

Comment:
Parsing a literal seems odd in the snippet showing people how to use the method. This is what I imagine people would usually want to do:
```py
cert = client.get_certificate("...")
parsed_id = parse_key_vault_certificate_id(cert.id)
```

Reply:
I like that!